lib/worker/batcher/message.ex

```diff
@@ -28,6 +28,15 @@ defmodule BorsNG.Worker.Batcher.Message do
     {"Delayed for higher-priority pull requests", :running}
   end
 
+  def generate_message({:preflight, :waiting}) do
+    ":clock1: Waiting for PR status (Github check) to be set, probably by CI. Bors will automatically try to run when all required PR statuses are set."
```

Comment:
Two changes I'd like to see here:
* Let's use GitHub's language: they call these things [status checks](https://help.github.com/en/github/administering-a-repository/enabling-required-status-checks)
* It'd be _great_ if this message explained how long Bors is configured to wait until erroring. We'd need access to `prerun_timeout_sec` - not sure if we can get that here easily.

```
    ":clock1: Bors is waiting for all GitHub status checks to complete and will continue polling until $TIME_IN_FUTURE
```